data/FoA_2022/metadata.yml

```diff
@@ -0,0 +1,212 @@
+source:
+  primary:
+    key: FOA_2022
+    bibtype: Online
+    year: 2022
+    author: Australian Biological Resources Study
+    title: Flora of Australia, Australian Biological Resources Study, Canberra.
+    url: http://www.ausflora.org.au
+contributors:
+  data_collectors:
+  - last_name: Coleman
+    given_name: David
+    ORCID: 0000-0002-1773-6597
+    affiliation: Evolution & Ecology Research Centre, School of Biological, Earth,
```

Comment:
Is David's affiliation consistent with ours?

Reply:
Yes, I copied what we had